io/include/pcl/compression/color_coding.h

```diff
@@ -281,12 +278,12 @@ class ColorCoding
     * \param rgba_offset_arg offset to color information
     */
   void
-  decodePoints (PointCloudPtr outputCloud_arg, std::size_t beginIdx_arg, std::size_t endIdx_arg, unsigned char rgba_offset_arg)
+  decodePoints (PointCloudPtr outputCloud_arg, index_t beginIdx_arg, index_t endIdx_arg, unsigned char rgba_offset_arg)
```

Comment:
Size_t is unsigned right? So could be uindex_t?

Reply:
My bad